nafas/functions.py

```diff
@@ -283,15 +283,19 @@ def play_sound(sound_path, enable=True):
         _ = nava.play(sound_path, async_mode=True)
 
 
-def run(program_data):
+def run(program_data, silent=False):
```

Comment:
All good, only thing is that I prefer to have `silent` inside `program_data` insted.

Reply:
`program_data` consists of program details such as `ratio`, `unit`, `pre`, and `cycle`. However, `silent` is related to running, so I believe it's not a good idea to inject `silent` to `program_data`.